docs/development_environment.mdx

```diff
@@ -46,7 +46,7 @@ When a task is currently running (like `Preview` for the docs), it can be restar
 
 ### Debugging with Visual Studio Code
 
-If the Dev Container was set up correctly - it supports debugging by default, out-of-the-box. It provides the necessary debug configurations, so hitting F5 should launch Home Assistant. Any breakpoints put in the code should be triggered, and the debugger should stop. 
+If the Dev Container was set up correctly - it supports debugging by default, out-of-the-box. It provides the necessary debug configurations, so hitting F5 should launch Home Assistant. Any breakpoints put in the code should be triggered, and the debugger should stop. If your dev instance started without issues, you should find the browser ui to the dev instance running on "localhost:8123".
```

Comment:
vscode should open (or offer) to open the UI.

Depending on how you are using it, it doesn't have to be available on the host/port you wrote.